domain-model/src/main/java/uk/gov/hmcts/cmc/domain/constraints/ValidAdmission.java

```diff
@@ -10,7 +10,7 @@
 
 @Target({ElementType.TYPE, ElementType.ANNOTATION_TYPE})
 @Retention(RetentionPolicy.RUNTIME)
-@Constraint(validatedBy = {ValidFullAdmissionConstraintValidator.class, ValidPartAdmissionConstraintValidator.class})
+@Constraint(validatedBy = {ValidPartAdmissionConstraintValidator.class})
```

Comment:
as we are using this for both full and part admission, suggest to rename `ValidPartAdmissionConstraintValidator` accordingly

Reply:
This change will get reverted when Patryk adds SoM validation for both full and part admissions.

Reply:
I'd rename this `ValidPartAdmissionConstraintValidator` to `ValidPartialAdmissionConstraintValidator`